tests/e2e/e2e_ibc_test.go

```diff
@@ -64,31 +64,24 @@ func (s *E2ETest) checkOutflows(umeeAPIEndpoint, denom string, checkWithExcRate
 }
 
 func (s *E2ETest) checkSupply(endpoint, ibcDenom string, amount math.Int) {
-	attempt := 1
 	s.Require().Eventually(
 		func() bool {
-			attempt++
 			supply, err := s.QueryTotalSupply(endpoint)
 			if err != nil {
 				return false
 			}
 			if supply.AmountOf(ibcDenom).Equal(amount) {
 				return true
 			}
-			if attempt == 59 {
-				// if we're nearing the end of our attempts, print expected and actual values
-				s.T().Logf("expected: %s, got: %s", amount, supply.AmountOf(ibcDenom))
-			}
 			return false
 		},
-		2*time.Minute,
+		10*time.Minute,
```

Comment:
why do we increase it to 10min? that sounds too much...

Reply:
We don't know ibc txn is succeed or not, we only know txn is succceed when that denom exists in supply query

Reply:
10 minutes I know it is too much but that is the maximum time to check the ibc txn because sometimes hermes relayer taking too much time submit the ibc txn on both sides